packages/oid4vci-demo-frontend/src/pages/SSIInformationManualRequestPage/index.tsx

```diff
@@ -13,13 +13,23 @@ import {useFlowRouter} from "../../router/flow-router"
 import {useEcosystem} from "../../ecosystem/ecosystem"
 import {useCredentialsReader} from "../../utils/credentials-helper"
 
+import {DEFAULT_FORM} from "../../types/default-form"
 
 type State = {
     data?: any
 }
 
-function getInitialState(form: DataFormRow[]) {
-    return transformFormConfigToEmptyObject(form)
+function getInitialState(formConfig: SSIInformationManualRequestPageConfig) {
+  if (!formConfig.form) {
+
+      formConfig.form = DEFAULT_FORM
+      return {
```

Comment:
if no form is in the config, it was discussed to not show anything. you just need to supply a form if you want 1

Reply:
fixed